tools/PrecompiledAbis.html

```diff
@@ -0,0 +1,394 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <meta charset='UTF-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    <script src='https://cdnjs.cloudflare.com/ajax/libs/web3/1.9.0/web3.min.js' integrity='sha512-mUvdtCaVKd9dby/UnYGKMLkP3eT7EA6QNFN5iSEjvMf8TFPq2oZFIemWL46iypQcL8xYNdx7wQIA1oYnTXuYhg==' crossorigin='anonymous' referrerpolicy='no-referrer'></script>
+    <link href='https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css' rel='stylesheet' integrity='sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN' crossorigin='anonymous'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-icons/1.11.2/font/bootstrap-icons.min.css" integrity="sha512-D1liES3uvDpPrgk7vXR/hR/sukGn7EtDWEyvpdLsyalQYq6v6YUsTUJmku7B4rcuQ21rf0UTksw2i/2Pdjbd3g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <title>Rootstock Bridge Methods Selectors And Events Topics</title>
+</head>
+
+<style>
+
+.bridge-button {
+    width: 540px;
+}
+
+#network-selection-container {
+    width: 720px;
+    margin: 8px;
+}
+
+.display {
+    overflow: scroll;
+}
+
+#custom-network-container {
+    display: none;
+}
+
+#fixed-network-container, #custom-network-container {
+    margin-top: 12px;
+    margin-bottom: 12px;
+}
+
+#customNetworkCheckbox {
+    cursor: pointer;
+}
+
+#customNetworkCheckboxLabel {
+    cursor: pointer;
+}
+
+</style>
+
+<body>
+
+    <h1 class='text-center'>Rootstock Bridge Methods Selectors and Events Topics</h3>
+    <h5 class='text-center'>Simple html page to get bridge methods selectors and events topics</h2>
+    
+    <div id='container'>
+
+        <div id="controlsContainer">
+            <div id="fixed-network-container" class="row justify-content-center align-items-center">
+                <div class="col-3">
+                    <b>Abis For Fork:</b>
+                </div>
+                <div class="col-9">
+                    <select id='abiForForkSelect' class='form-select form-select-sm' aria-label=''>
+                        <option selected value='fingerroot'>Fingerroot</option>
+                        <option value='hop'>Hop</option>
+                        <option value='iris_4_0_2'>Iris 4.0.2</option>
```

Comment:
IRIS is actually version 3 of rskj, HOP is version 4.

And is there any difference in the Bridge abi between IRIS 4.0.0 and IRIS 4.0.2? If not I think it only adds a bit of confusion here

Reply:
Updated!